stdlib/public/TensorFlow/Ops.swift

```diff
@@ -686,6 +686,11 @@ public extension Tensor {
   static func ++ (lhs: Tensor, rhs: Tensor) -> Tensor {
     return lhs.concatenated(with: rhs)
   }
+
+  /// Returns a concatenated tensor of the given tensors.
+  init(concatenating tensors: [Tensor<Scalar>]) {
```

Comment:
For generality, this should take an array of indices (if concatV2 takes multidimensional indices) with label ‘alongAxes:’, defaulted to 0.